motionplan/planManager.go

```diff
@@ -654,17 +659,8 @@ func (pm *planManager) goodPlan(pr *rrtPlanReturn, opt *plannerOptions) (bool, f
 	return false, solutionCost
 }
 
-func (pm *planManager) planToRRTGoalMap(plan Plan, goal spatialmath.Pose) (*rrtMaps, error) {
-	planNodes := make([]node, 0, len(plan))
-	// Build a list of nodes from the plan
-	for _, planStep := range plan {
-		conf, err := pm.frame.mapToSlice(planStep)
-		if err != nil {
-			return nil, err
-		}
-		planNodes = append(planNodes, newConfigurationNode(conf))
-	}
-
+func (pm *planManager) planToRRTGoalMap(plan *rrtPlan, goal spatialmath.Pose) (*rrtMaps, error) {
+	planNodes := plan.nodes
```

Comment:
This would be the reason to prefer rrtPlan, but reverting to the above isnt that bad